s3/pkg/conf/tidb.sql

```diff
@@ -0,0 +1,251 @@
+
+--
+-- Table structure for table `gcpart`
+--
+
+DROP TABLE IF EXISTS `gcpart`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `gcpart` (
+  `partnumber` int(11) DEFAULT NULL,
+  `size` bigint(20) DEFAULT NULL,
+  `objectid` varchar(255) DEFAULT NULL,
+  `offset` bigint(20) DEFAULT NULL,
+  `etag` varchar(255) DEFAULT NULL,
+  `lastmodified` datetime DEFAULT NULL,
+  `initializationvector` blob DEFAULT NULL,
+  `bucketname` varchar(255) DEFAULT NULL,
+  `objectname` varchar(255) DEFAULT NULL,
+  `version` bigint(20) UNSIGNED DEFAULT NULL,
+   KEY `rowkey` (`bucketname`,`objectname`,`version`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `multipartpart`
+--
+
+DROP TABLE IF EXISTS `multipartpart`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `multipartpart` (
+  `partnumber` int(11) DEFAULT NULL,
+  `size` bigint(20) DEFAULT NULL,
+  `objectid` varchar(255) DEFAULT NULL,
+  `offset` bigint(20) DEFAULT NULL,
+  `etag` varchar(255) DEFAULT NULL,
+  `lastmodified` datetime DEFAULT NULL,
+  `initializationvector` blob DEFAULT NULL,
+  `bucketname` varchar(255) DEFAULT NULL,
+  `objectname` varchar(255) DEFAULT NULL,
+  `uploadtime` bigint(20) UNSIGNED DEFAULT NULL,
+   KEY `rowkey` (`bucketname`,`objectname`,`uploadtime`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `multiparts`
+--
+
+DROP TABLE IF EXISTS `multiparts`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `multiparts` (
+  `bucketname` varchar(255) DEFAULT NULL,
+  `objectname` varchar(255) DEFAULT NULL,
+  `uploadtime` bigint(20) UNSIGNED DEFAULT NULL,
+  `initiatorid` varchar(255) DEFAULT NULL,
+  `ownerid` varchar(255) DEFAULT NULL,
+  `contenttype` varchar(255) DEFAULT NULL,
+  `location` varchar(255) DEFAULT NULL,
+  `pool` varchar(255) DEFAULT NULL,
+  `acl` JSON DEFAULT NULL,
+  `sserequest` JSON DEFAULT NULL,
+  `encryption` blob DEFAULT NULL,
+  `attrs` JSON DEFAULT NULL,
+  `storageclass` tinyint(1) DEFAULT 0,
+  UNIQUE KEY `rowkey` (`bucketname`,`objectname`,`uploadtime`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `objectpart`
+--
+
+DROP TABLE IF EXISTS `objectpart`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `objectpart` (
+  `partnumber` int(11) DEFAULT NULL,
+  `size` bigint(20) DEFAULT NULL,
+  `objectid` varchar(255) DEFAULT NULL,
+  `offset` bigint(20) DEFAULT NULL,
+  `etag` varchar(255) DEFAULT NULL,
+  `lastmodified` datetime DEFAULT NULL,
```

Comment:
please change ALL time fields to support date and time with timezone

Reply:
@rhsakarpos  I have add a issue to track this.   #681